.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+-   id: byexample
+    name: run code snippets in docstrings to validate them
```

Comment:
We could rephrase this as `"Run code snippets to validate them"`. I'm not 100% sure of my own proposal but I think that the word `"docstring"` suggests that it will only work for Python's docstrings which it's only one option.

What do you think?

Reply:
Indeed, you are correct. I am most familiar with Python’s docstrings and I did not think of the fact that not all languages put their docs into strings or use this term :)